toml/src/main/java/com/electronwill/nightconfig/toml/ArrayWriter.java

```diff
@@ -44,6 +44,7 @@ static void write(List<?> values, CharacterOutput output, TomlWriter writer) {
 			writer.decreaseIndentLevel();
 			writer.writeNewline(output);
 		}
+		writer.writeIndent(output);
```

Comment:
```suggestion
		
		writer.writeIndent(output);
		}
```
it should be in the if condition, to respect the "enable/disable indent" flag